### ISSUE TITLE
64bit is not the default

### ISSUE THREAD

**Issue by pmatos** [esbmc/esbmc#129]
ESBMC 6.1.0.

Even though it says --64 is the default, I get different behaviours when using it.

Take 

`fact.c`:
```
#include <stdio.h>

unsigned int fact (unsigned int n) {
  unsigned int f = 1;
  while (n != 0)
      f *= n--;

  return f;
}
```

`fact_props.c`:
```
#include "fact.c"


void check_fact (void)
{
  unsigned int n = nondet_uint ();
  unsigned int f = fact (n);

  assert (f > n);
}
```

Run:
```
~/installs/esbmc+python-v6.1.0-linux-static-64/bin/esbmc fact_props.c --function check_fact --k-induction-parallel --boolector 
ESBMC version 6.1.0 64-bit x86_64 linux
file fact_props.c: Parsing
file fact_props.c: Parsing
file fact_props.c: Parsing
In file included from fact_props.c:1:
In file included from ./fact.c:1:
In file included from /usr/include/stdio.h:27:
In file included from /usr/include/bits/libc-header-start.h:33:
In file included from /usr/include/features.h:474:
/usr/include/gnu/stubs.h:7:11: fatal error: 'gnu/stubs-32.h' file not found
# include <gnu/stubs-32.h>
          ^~~~~~~~~~~~~~~~
In file included from fact_props.c:1:
In file included from ./fact.c:1:
In file included from /usr/include/stdio.h:27:
In file included from /usr/include/bits/libc-header-start.h:33:
In file included from /usr/include/features.h:474:
/usr/include/gnu/stubs.h:7:11: fatal error: 'gnu/stubs-32.h' file not found
# include <gnu/stubs-32.h>
          ^~~~~~~~~~~~~~~~
PARSING ERROR
PARSING ERROR
In file included from fact_props.c:1:
In file included from ./fact.c:1:
In file included from /usr/include/stdio.h:27:
In file included from /usr/include/bits/libc-header-start.h:33:
In file included from /usr/include/features.h:474:
/usr/include/gnu/stubs.h:7:11: fatal error: 'gnu/stubs-32.h' file not found
# include <gnu/stubs-32.h>
          ^~~~~~~~~~~~~~~~
PARSING ERROR
**** WARNING: Base case process crashed.
**** WARNING: Forward condition process crashed.
**** WARNING: Inductive step process crashed.
Message from unrecognized k-induction child process
[1]    3631 abort (core dumped)  ~/installs/esbmc+python-v6.1.0-linux-static-64/bin/esbmc fact_props.c
```

Now try with `--64` in the command line.

**Comment by mikhailramalho** [comment]
Thank you for the report!

PR#131 should address it.

**Comment by mikhailramalho** [comment]
Fixed by PR#131.
